scripts/createReleasePipeline/bashScripts/createResources.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+
+#Create App Service Plan
+az appservice plan create --resource-group $1 \
+    --name $2-plan \
+    --is-linux \
+    --sku B1
+
+#Create Webapp
+az webapp create --resource-group $1 \
+    --plan $2-plan \
+    --name $2 \
+    --runtime "python|2.7"
+
+#Create App Insights
+az resource create \
+    --resource-group $1 \
+    --resource-type "Microsoft.Insights/components" \
+    --name $2-appinsights \
+    --location eastus\
```

Comment:
lets make note at some point we might want to parameterize this setting

Reply:
I am not sure we need to parameterize the "Microsoft.Insights/components"  and the eastus is hard coded because app insights does not work in centralus. Microsoft does not like fly over states....